brianlong/www.validators.app
# 178962742_eager_load_vote_accounts

#### What's this PR do?
- Includes vote_accounts

#### How should this be manually tested?
- Make a request to ie. https://stage.validators.app/api/v1/validators/testnet.json (or localhost), check response time

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/178962742

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
